starwars/src/components/StarwarsData.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Character from './Character';
+import './StarWars.css';
+
+function StarwarsData(props) {
+    return (
+        <div className="character-cards">
+            {props.charInfoList.map(( element, index ) => <Character key={element.created} traits={element} />)}
```

Comment:
Since you're not using the index argument in .map, you can remove it. It's an optional argument. I love that you used `key={element.created}`. Very smart!

Reply:
@gooseandmegander Oh right, I could've deleted that. Originally I placed index as the Key, but then decided to use the created date!